### PR TITLE
Update to 0.9.0

### PR DESCRIPTION
**Pull request by blasten**
- Update to Polymer 0.9.0. and add tests

### CODE REVIEW

File: demo/demo1.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+-->
+<html>
+<head>
+  <title>paper-scroll-header-panel: demo1</title>
+
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
+  <meta name="mobile-web-app-capable" content="yes">
+  <meta name="apple-mobile-web-app-capable" content="yes">
+
+  <script src="../../webcomponentsjs/webcomponents.js"></script>
```
Comment (cdata) [inline demo/demo1.html]: `webcomponents-lite.js`

File: demo/demo2.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+-->
+<html>
+<head>
+  <title>paper-scroll-header-panel: demo2</title>
+
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+  <meta name="mobile-web-app-capable" content="yes">
+  <meta name="apple-mobile-web-app-capable" content="yes">
+
+  <script src="../../webcomponentsjs/webcomponents.js"></script>
```
Comment (cdata) [inline demo/demo2.html]: `webcomponents-lite.js`

File: demo/demo3.html [hunk 3]
```diff
@@ -17,41 +18,37 @@
 
   <script src="../../webcomponentsjs/webcomponents.js"></script>
```
Comment (cdata) [inline demo/demo3.html]: `webcomponents-lite.js`

File: demo/demo4.html [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+-->
+<html>
+<head>
+  <title>paper-scroll-header-panel: demo4</title>
+
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
+  <meta name="mobile-web-app-capable" content="yes">
+  <meta name="apple-mobile-web-app-capable" content="yes">
+
+  <script src="../../webcomponentsjs/webcomponents.js"></script>
```
Comment (cdata) [inline demo/demo4.html]: `webcomponents-lite.js`

File: demo/demo5.html [hunk 5]
```diff
@@ -17,50 +18,46 @@
 
   <script src="../../webcomponentsjs/webcomponents.js"></script>
```
Comment (cdata) [inline demo/demo5.html]: `webcomponents-lite.js`

**Comment by blasten** [comment]
can you take another look @cdata ?

File: core-scroll-header-panel.html [hunk 6]
```diff
@@ -1,360 +0,0 @@
...
-      <div>Content goes here...</div>
-    </core-scroll-header-panel>
-
-@group Polymer Core Elements
-@element core-scroll-header-panel
-@mixins Polymer.CoreResizable https://github.com/polymer/core-resizable
-@homepage github.io
--->
-
-<link rel="import" href="../polymer/polymer.html">
```
Comment (atotic) [inline core-scroll-header-panel.html]: docs for the element are not showing up. Docs block should be right before dom-module. Move <links> above the docs.

File: paper-scroll-header-panel.html [hunk 7]
```diff
@@ -0,0 +1,450 @@
...
+  paper-scroll-header-panel {
+    --paper-scroll-header-panel-condensed-header: {
+      background-color: #f4b400;
+    };
+  }
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../iron-resizable-behavior/iron-resizable-behavior.html">
+
```
Comment (atotic) [inline paper-scroll-header-panel.html]: Docs are not showing up. <!-- documentation -> must be right above dom-module. Move <link> elements above the docs.

File: test/basic.html [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+
+      test('condensation', function(done) {
+        var top1 = toolbar.getBoundingClientRect().top;
+
+        scrollHeaderPanel.condenses = true;
+        scrollHeaderPanel.headerHeight = 150;
+        scrollHeaderPanel.condensedHeaderHeight = 50;
+        scrollHeaderPanel.scroller.scrollTop = 300;
+
+        asyncPlatformFlush(function() {
```
Comment (atotic) [inline test/basic.html]: asyncPlatformFlush depreciated. Replace with flush()

**Comment by atotic** [comment]
Minor fixes, otherwise good.

**Comment by atotic** [comment]
How does one navigate betwee demo/index.html to demo/demoX.html

File: paper-scroll-header-panel.html (hunk 7 above)
Comment (blasten) [inline paper-scroll-header-panel.html]: Moved!

File: test/basic.html (hunk 8 above)
Comment (blasten) [inline test/basic.html]: :+1: Replaced. I forgot about this one.

**Comment by blasten** [comment]
@atotic could you take another look? There isn't a way to navigate between the demos. I think I could address this issue in a new PR.

**Comment by atotic** [comment]
LGTM. @cdata, want to merge?

**Pull request merged**